packages/plugins/color-picker/admin/src/index.js

```diff
@@ -47,6 +48,7 @@ export default {
               {
                 name: 'required',
                 type: 'checkbox',
+                defaultValue: true,
```

Comment:
I think it should be false by default, since that's how all normal fields (not slug or uid) work

Reply:
ok yeah I think you are right it doesn't make sense for this to default to true